### ISSUE TITLE
Is there support for github enterprise

### ISSUE THREAD

**Issue by nick4eva** [dlvhdr/gh-dash#285]
**Is your feature request related to a problem? Please describe.**
I'd like to have github enterprise support so I can use this dashboard for my work repos

**Describe the solution you'd like**
I'd like to have an option in `config.yml` to specify which github server to use (regular one or github enterprise)

**![image](https://github.com/dlvhdr/gh-dash/assets/136439/ab585ed5-3241-4eb4-a1b0-b06e4df56dfd)**
I've tried just map my work repos to my work org, but obviously that didn't work since `gh-dash` doesn't know address of our github enterprise server and have no personal access token to read data

**Additional context**

**Comment by dlvhdr** [comment]
I'll be happy to get contributions for this. 
I dont have a github enterprise account so I have no idea how is it different from a regular one. Also, I wouldnt be able to test this.

**Comment by nick4eva** [comment]
I've realized that it will work if I'll pass the `GH_HOST` and `GH_ENTERPRISE_TOKEN` env vars before running `gh-dash` (because `go-gh` package is used under the hood which is recognizing those env vars). 
And it works actually 🎉 
Not sure though how to make it to use those host and token permanently, currently I need to set up them each time I've started shell.
Also, facing issues with some missing icons (not sure if that's related to the fact that I'm using Nerd Fonts)
![image](https://github.com/dlvhdr/gh-dash/assets/136439/6710edf4-b19f-4655-a4a8-5de6319e0c1c)
![image](https://github.com/dlvhdr/gh-dash/assets/136439/f73c8779-2054-4a7f-89f2-c28804d6c4b2)

**Comment by donbex** [comment]
I don't think this requires support at the extension level. With the [`gh auth login`](https://cli.github.com/manual/gh_auth_login) command you can already log into a GHE account and securely store the credentials, so that all later `gh` commands will refer to that GHE account, including `gh dash`.

**Comment by danriedl** [comment]
I use a alias and a env variable:
My `.zshrc`:
```bash
#GitHub
alias ghe='GH_HOST=ourgithub.enterprise.com gh'
export GH_ENTERPRISE_TOKEN="$(cat $HOME/.git_token_enterprise)"
```

so by calling `gh` i refer to github.com and with `ghe` to the internal one.

**Comment by xamcost** [comment]
Indeed, as @T3sT3ro mentioned, I can confirm that if you set `GH_HOST` in your env, it works perfectly with an entreprise account.
